Hexastore.Web/Queue/QueueWriter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Hexastore.Web.EventHubs;
+using Microsoft.Extensions.Logging;
+
+namespace Hexastore.Web.Queue
+{
+    public class QueueWriter : IDisposable
+    {
+        private readonly BlockingCollection<StoreEvent> _queue;
+        private readonly EventReceiver _eventReceiver;
+        private readonly ILogger _logger;
+        private readonly Task _task;
+        private readonly CancellationTokenSource _cts = new CancellationTokenSource();
+
+        public QueueWriter(EventReceiver eventReceiver, ILogger logger)
+        {
+            _queue = new BlockingCollection<StoreEvent>();
+            _eventReceiver = eventReceiver;
+            _logger = logger;
+            _task = StartReader();
+            _task.Start();
```

Comment:
nit: I would do a Task.Run here instead of returning a new task inside Start

Reply:
done